pom.xml

```diff
@@ -84,8 +84,9 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.222.x</artifactId>
-        <version>26</version>
+        <artifactId>bom-2.249.x</artifactId>
+        <!-- TODO weekly -->
+        <version>966.v3857b7c82032</version>
```

Comment:
@jglick - same question as on https://github.com/jenkinsci/google-storage-plugin/pull/172/files#r758540082. Thanks!

Reply:
I did a similar change in https://github.com/jenkinsci/antisamy-markup-formatter-plugin/blob/master/pom.xml#L49 quite recently @abayer in case that can help :)

Reply:
Due to https://github.com/jenkinsci/google-oauth-plugin/blob/c34929cdf388801fe4eb2a0cc7e183c987f7bc7e/pom.xml#L70 you need to use `bom-weekly`. Current choices: https://github.com/jenkinsci/bom/blob/6f1d049ca66c1ede0ce185b88ac8aa30c5e16947/pom.xml#L33-L36